jerry-core/ecma/base/ecma-gc.c

```diff
@@ -1345,8 +1351,9 @@ ecma_gc_free_properties (ecma_object_t *object_p) /**< object */
           break;
         }
 #endif /* ENABLED (JERRY_ESNEXT) */
-#if ENABLED (JERRY_BUILTIN_WEAKMAP) || ENABLED (JERRY_BUILTIN_WEAKSET)
+#if ENABLED (JERRY_BUILTIN_WEAKMAP) || ENABLED (JERRY_BUILTIN_WEAKSET || ENABLED (JERRY_BUILTIN_WEAKREF))
```

Comment:
`#if ENABLED (JERRY_BUILTIN_WEAKMAP) || ENABLED (JERRY_BUILTIN_WEAKSET) || ENABLED (JERRY_BUILTIN_WEAKREF)` (wrong order in parenthesis)